pkgs/applications/video/lbry/default.nix

```
@@ -29,7 +29,7 @@ in appimageTools.wrapAppImage rec {
     # Now, install assets such as the desktop file and icons
     install -m 444 -D ${src}/lbry.desktop -t $out/share/applications
     substituteInPlace $out/share/applications/lbry.desktop \
-      --replace 'AppRun' '$out/bin/lbry'
+      --replace 'AppRun' $out/bin/lbry
```

If you do a simple grep for AppRun over nixpkgs, you'll see most of the usage looks like this:

```
--replace 'Exec=AppRun' 'Exec=${pname}'
```

since we can't use `pname` here, because it differs, I suggest to just use this:

```suggestion
      --replace 'Exec=AppRun' 'Exec=lbry'
```